client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -210,74 +210,79 @@ export const RequestLineEditForm = ({
-            Input={
-              <NumericTextInput
-                autoFocus
-                value={numberOfPacksFromQuantity(requestedQuantity)}
-                disabled={isPacks}
-                width={100}
-                onChange={q =>
-                  update({
-                    requestedQuantity: q && numberOfPacksToTotalQuantity(q),
-                  })
-                }
-              />
-            }
-            labelWidth="750px"
-            label={t('label.order-quantity')}
-          />
-          {isPacksEnabled && (
+            )}
             <InputWithLabelRow
               Input={
                 <NumericTextInput
-                  autoFocus
-                  disabled={!isPacks}
-                  value={NumUtils.round(
-                    requestedQuantity / currentItem.defaultPackSize,
-                    2
+                  width={100}
+                  value={numberOfPacksFromQuantity(
+                    draftLine?.suggestedQuantity ?? 0
                   )}
+                  disabled
+                />
+              }
+              labelWidth="750px"
+              label={t('label.suggested-quantity')}
+            />
+            <InputWithLabelRow
+              Input={
+                <NumericTextInput
+                  autoFocus
+                  value={numberOfPacksFromQuantity(requestedQuantity)}
+                  disabled={isPacks}
                   width={100}
-                  onChange={quantity => {
-                    if (quantity === undefined) return;
+                  onChange={q =>
                     update({
-                      requestedQuantity: quantity * currentItem.defaultPackSize,
-                    });
-                  }}
+                      requestedQuantity: q && numberOfPacksToTotalQuantity(q),
+                    })
+                  }
                 />
               }
               labelWidth="750px"
-              label={t('label.requested-packs')}
+              label={t('label.order-quantity')}
             />
-          )}
-        </>
+            {isPacksEnabled && (
+              <InputWithLabelRow
+                Input={
+                  <NumericTextInput
+                    autoFocus
+                    disabled={!isPacks}
+                    value={NumUtils.round(
+                      requestedQuantity / currentItem.defaultPackSize,
+                      2
+                    )}
+                    width={100}
+                    onChange={quantity => {
+                      if (quantity === undefined) return;
+                      update({
+                        requestedQuantity:
+                          quantity * currentItem.defaultPackSize,
+                      });
+                    }}
+                  />
+                }
+                labelWidth="750px"
+                label={t('label.requested-packs')}
+              />
+            )}
+          </>
+        ) : (
+          <></>
+        )
```

Comment:
```suggestion
  ) : null
```

Prefer this to empty tags personally :)

Reply:
Oh, you have to also expand the type: `Middle: React.ReactElement | null;`

Seems reasonable though, since `null` is the standard way to render empty React elements.